### PR TITLE
Fix pod lint issues

### PR DESCRIPTION
**Pull request by frederoni**
Fixes pod lint issues and bumps the version

@bsudekum @1ec5 👀

### CODE REVIEW

File: turf-swift.podspec [hunk 1]
```diff
@@ -6,8 +6,8 @@ Pod::Spec.new do |s|
   s.version = "0.0.3"
```
Comment (1ec5) [inline turf-swift.podspec]: [The marketing version in Info.plist](https://github.com/mapbox/turf-swift/blob/master/Turf/Info.plist#L18) should match this version.

**Comment by 1ec5** [comment]
FYI, publishing this pod to CocoaPods trunk will lock in the `turf-swift` name and module name. That’s OK because it leaves open the possibility of an Objective-C-specific pod in the future.

**Comment by frederoni** [comment]
Considering the wrappers we've talked about in #2, a more generic name would make sense.
Would just changing the pod name in the podspec be sufficient?

**Comment by 1ec5** [comment]
> Would just changing the pod name in the podspec be sufficient?

Yep, that’s all that CocoaPods cares about – not even the podspec’s file name matters.

**Pull request merged**